README.md

```diff
@@ -96,43 +96,41 @@ For a list of all available commands run `npm run`.
 These commands are
 
 * building
-  * `npm run build` creates symlinks among packages and creates node, umd, and esm outputs for each package
+  * `npm run build` Creates esm and cjs outputs for each package
+  * `npm run build:esm` Creates esm outputs for each package
+  * `npm run build:cjs` Creates cjs outputs for each package
   * `npm run clean` runs `clean:src` and `clean:dist` _(requires bash console)_
     * `npm run clean:src` deletes `.d.ts`, `.js`, and `.js.map` files
     * `npm run clean:dist` deletes `.rpt2_cache` and `dist` folders
+  * `npm run deepclean` deletes node_modules and runs `clean`
   * `npm run lint` lints the TypeScript files
-  * `npm run lint:fix` lints the TypeScript files and fixes
+  * `npm run lint:fix` lints the TypeScript files and fixes what it can
   * `npm run prettify` beautifies TypeScript files
+  * `npm run getupdates` modifies package.json with npm updates to packages
 
 * testing
   * `npm run test` lints, then runs `test:chrome` tests to confirm that the API is functioning as expected
-  * `npm run test:browsers` runs karma in the Chrome, Firefox, and Chromium Edge browsers
   * `npm run test:chrome` runs karma in the Chrome browser
   * `npm run test:chrome:ci` runs karma in the ChromeHeadlessCI browser
   * `npm run test:chrome:debug` runs karma in the Chrome browser and leaves the browser open for debugging tests
-  * `npm run test:edge` runs karma in the Edge (Chromium) browser
   * `npm run test:firefox` runs karma in the Firefox browser
-  * `npm run test:ci` lints, then runs `test:chrome:ci`, `test:firefox`, and `coveralls` from a bash window
-  * `npm run test:ci:win` lints, then runs `test:chrome:ci`, `test:firefox`, and `coveralls:win` from a Windows window
-  * `npm run test:all` runs `test:chrome` and `test:edge` and `test:firefox`
-  * `npm run coveralls` updates code coverage info from a bash window
-  * `npm run coveralls:win` updates code coverage info from a Windows window
+  * `npm run test:firefox:ci` runs karma in the FirefoxHeadless browser
+  * `npm run test:all` runs `test:chrome` and `test:firefox`
 
 * publishing doc
   * `npm run docs:build` builds the documentation ___(note that this script creates a `docs` folder, deleting any existing one)___
   * `npm run docs:deploy` pushes the documentation to the repository's gh-pages
+  * `npm run docs:srihash` generates the srihashes
```

Comment:
`npm run docs:srihash` generates the srihashes.  Learn more about SRI Hash here https://www.srihash.org/
 
 I had no idea what SRI hash was, so figure a link to it would be good.